nextest-runner/src/runner/executor.rs

```diff
@@ -768,7 +768,7 @@ impl<'a> ExecutorContext<'a> {
         // Use an event that happens shortly after the test starts to avoid
         // overwhelming progress bar display code.
         let mut show_progress_sleep =
-            std::pin::pin!(crate::time::pausable_sleep(Duration::from_millis(100)));
+            std::pin::pin!(crate::time::pausable_sleep(Duration::from_millis(10)));
```

Comment:
Also shortened this timeout -- 10ms is long enough that rapid-fire tests won't spam output, but is also less than 1 frame on 60fps.